core/jazz_services/config/dev-config.json

```diff
@@ -3,8 +3,8 @@
     "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status"],
     "service_update_fields": ["description", "email", "slack_channel", "tags", "repository", "status","metadata","is_public_endpoint"],
     "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status"],
-    "service_optional_fields": ["description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint"],
-    "service_return_fields": ["service", "domain", "type", "created_by", "runtime", "description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint"],
+    "service_optional_fields": ["description", "region", "repository", "email", "slack_channel", "tags", "status","metadata","is_public_endpoint", "deployment_targets"],
```

Comment:
`deployment_targets` should be in `service_required_fields`. Please update other configurations & unit tests if required.

Reply:
fixed